test/server-test.js

```diff
@@ -23,5 +25,45 @@ describe('Server', function(){
     assert(app)
   })
 
-  
+  describe('GET /api/foods', function() {
+    this.timeout(1000000)
+    beforeEach(function(done) {
+      Food.createFood("Banana", 105)
+      .then(function() {
+        Food.createFood("French Silk Pie", 340).then(function() { done() })
+      });
+    })
+
+    afterEach(function(done) {
+      Food.emptyFoodsTable().then(function() {
+        done()
+      })
+    })
+
+    it('should have two food items from the resource', function(done) {
+      var ourRequest = this.request
+      Food.findAllFoods().then(function(data) {
```

Comment:
i'd be okay with you hardcoding this test to avoid you needing to query the database again. that way, you just need to make the request and check the return value. 

that being said, i think what you have here is fine and you're on the right track!